docs/auth/social.mdx

```diff
@@ -127,7 +127,7 @@ Install the [`flutter_facebook_auth`](https://pub.dev/packages/flutter_facebook_
 
 ```yaml title="pubspec.yaml"
 dependencies:
-  flutter_facebook_auth: "{{ external.flutter_facebook_auth }}"
+  flutter_facebook_auth: ^0.3.0
```

Comment:
We keep a source of variables here: https://github.com/FirebaseExtended/flutterfire/blob/master/docs/versions.js - could you update that file rather than this one?

Thanks!